<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>moniteur</class>
 <widget class="QWidget" name="moniteur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1182</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>193</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_new_client">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton
{
background-color: transparent ;
border : none ;

}
QToolButton::hover
{
    border-style: outset;
    border-width: 2px;
    border-color: white;
}
	</string>
          </property>
          <property name="text">
           <string>Nouveau 
moniteur</string>
          </property>
          <property name="icon">
           <iconset resource="../ressources.qrc">
            <normaloff>:/Assets/Images/icons/Xornet's icons/add_employee.ico</normaloff>:/Assets/Images/icons/Xornet's icons/add_employee.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_viewProfile">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton
{
background-color: transparent ;
border : none ;

}
QToolButton::hover
{
    border-style: outset;
    border-width: 2px;
    border-color: white;
}
	</string>
          </property>
          <property name="text">
           <string>voir profile</string>
          </property>
          <property name="icon">
           <iconset resource="../ressources.qrc">
            <normaloff>:/Assets/Images/icons/profile2.ico</normaloff>:/Assets/Images/icons/profile2.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>280</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="TableViewTools" name="widget" native="true"/>
       </item>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableViewTools</class>
   <extends>QWidget</extends>
   <header>tableviewtools.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
